docs/advanced/document-store/benchmark.md

```diff
@@ -169,7 +336,7 @@ and report quality measurement (`Recall@K`).
 * **Not just speed, but also quality**: Since some backends offer better performance and some offer better quality, we make sure to report the quality measurement for the approximate nearest neighbor search. This will allow users to 
 choose the backend that best suits their cases.
 * **Same experiment, same API**: DocArray offers the same API across all backends and therefore we built on top of it the 
-same benchmarking experiment. Furthermore, we made sure to fix the same HNSW parameters for backends that support 
+same benchmarking experiment. Furthermore, we made sure to run the experiment with a series of HNSW parameters for backends that support 
 approximate nearest neighbor search. All backends are run on official Docker containers, local to the DocArray client 
 which allows having similar network overhead. We also allocate the same resources for those Docker containers and all 
 servers are run in a single node setup.
```

Comment:
I'm referring to  the text below:

> [raise a PR or issue](https://github.com/jina-ai/docarray/issues/new/choose).
let's change this link to the new docarray repository